code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -75,6 +75,7 @@ var/list/ai_verbs_default = list(
 	var/APU_power = 0							// If set to 1 AI runs on APU power
 	var/hacking = 0								// Set to 1 if AI is hacking APC, cyborg, other AI, or running system override.
 	var/system_override = 0						// Set to 1 if system override is initiated, 2 if succeeded.
+	var/synthetic_takeover = 0					// 1 is started, 2 is complete.

```

Comment:
Yeh, this would be a fine place for that list. Since all the rest of the malf related vars are already here.